Automattic/sensei
# Fix CSS code displaying in lesson template

Resolves #6731.

## Proposed Changes
This was a tricky / weird one! 😅 

When first navigating to the editor and selecting the Lesson template, the element immediately following `sensei-course-theme__header` is a `style` tag. As a result, [this CSS](https://github.com/Automattic/sensei/pull/6758/files#diff-4c801507864cb7e80a436e72964c376785a8d887025264d3f33eff586065013dL52) was being applied to that element. This was the root cause of the issue.

The issue resolved itself after refreshing the page because, after the refresh, the `sensei-course-theme__columns` is now the element that appears immediately after `sensei-course-theme__header`, so it gets the CSS applied instead (incidentally, it does nothing as there is editor-specific CSS that overrides it, so it only applies on the frontend).

Note that this is only a problem on WordPress 6.2.

## Testing Instructions


1. Activate the Course theme.
2. Go to Appearance > Editor > Templates.
3. Select the Lesson template.
4. Ensure no CSS code appears in the preview window (see screenshot in original issue).
5. Edit the template.
6. Ensure no CSS code appears in the editor.
7. Refresh the page.
8. Nothing should change.
9. Browse to a lesson on the frontend.
10. Ensure the `sensei-course-theme__columns` element has a top margin of `116px`.

## Pre-Merge Checklist
